targets/TARGET_STM/TARGET_STM32F4/common_objects.h

```diff
@@ -84,17 +84,28 @@ struct spi_s {
 };
 
 struct i2c_s {
+    /*  The 1st 2 members I2CName i2c
```

Comment:
OK let me know once done . I'll go through the files once again, trigger CI afterwards